tests/test_schemas.py

```diff
@@ -951,3 +953,36 @@ def custom_action(self, request, pk):
 
     assert inspector.should_include_endpoint(path, callback)
     assert inspector.get_allowed_methods(callback) == ["GET"]
+
+
+@unittest.skipUnless(coreapi, 'coreapi is not installed')
+@pytest.mark.parametrize(
+    ('method', 'expected'), [
+        ('get', True),
+        ('put', True),
+        ('patch', True),
+        ('delete', True),
+        ('GET', True),
+        ('PUT', True),
+        ('PATCH', True),
+        ('DELETE', True),
+        ('foo', False),
+        ('FOO', False),
+    ]
+)
```

Comment:
idk why parameterize is failing on py2k. I'd consider changing this to a class with a few test methods instead of parameterization.